### PR TITLE
Write Build Logs after `ci`

### PR DESCRIPTION
**Pull request by paulcichonski**
Best reviewed by reading commits one-by-one.

Implements #43

### CODE REVIEW

**Comment by paulcichonski** [comment]
hrm, this seems to introduce a panic every few runs:

```
goroutine 1 [running]:
panic(0x3d0700, 0xc4202963e0)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
main.panicOnError.func1(0xc42008cc80)
	/go/src/github.com/cisco/elsy/main/commands.go:382 +0x19e
github.com/cisco/elsy/vendor/github.com/codegangsta/cli.Command.Run(0x3eca83, 0x2, 0x0, 0x0, 0x0, 0x0, 0x0, 0x401bb5, 0x4f, 0x0, ...)
	/go/src/github.com/cisco/elsy/vendor/github.com/codegangsta/cli/command.go:179 +0xbbe
github.com/cisco/elsy/vendor/github.com/codegangsta/cli.(*App).Run(0xc4200889a0, 0xc42000a1b0, 0x3, 0x3, 0x0, 0x0)
	/go/src/github.com/cisco/elsy/vendor/github.com/codegangsta/cli/app.go:196 +0xc00
github.com/cisco/elsy/vendor/github.com/codegangsta/cli.(*App).RunAndExitOnError(0xc4200889a0)
	/go/src/github.com/cisco/elsy/vendor/github.com/codegangsta/cli/app.go:207 +0x53
main.main()
	/go/src/github.com/cisco/elsy/main/main.go:51 +0x167
```

Need to dig into that, so I guess hold off on reviewing for now.

**Comment by paulcichonski** [comment]
squashed in small fixup to 2d8ee0f to ensure bbt test containers do not exit (was causing race-condition where docker-compose would sometimes fail to link in services to `test` if they had already finished their echo.)

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@
 vendor/*/
 
 test-reports/
+elsy_build_logs/
```
Comment (joeygibson) [inline .gitignore]: Could we just call the directory `build_logs`? We've really only been using `elsy` for searchability.

Comment (paulcichonski) [inline .gitignore]: sure, updated to use `build_logs`: ac0cfbc

**Pull request merged**